CHANGELOG.md

```diff
@@ -3,6 +3,22 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [0.8.1-alpha.0](https://github.com/chanzuckerberg/edu-design-system/compare/v0.6.0...v0.8.1-alpha.0) (2022-03-02)
+
+
+### Features
+
+* add ButtonGroup component ([#808](https://github.com/chanzuckerberg/edu-design-system/issues/808)) ([c413ac6](https://github.com/chanzuckerberg/edu-design-system/commit/c413ac674479919c2931e7dd03b5d5f08d18b43a))
+* add Dropdown component ([#835](https://github.com/chanzuckerberg/edu-design-system/issues/835)) ([acf54dc](https://github.com/chanzuckerberg/edu-design-system/commit/acf54dca68b784ca6718cc16927b7a97d55db6b8))
+* add DropdownButton ([#804](https://github.com/chanzuckerberg/edu-design-system/issues/804)) ([8b2d49f](https://github.com/chanzuckerberg/edu-design-system/commit/8b2d49f442e825e2e14ff8ad403f5a873edd5b8b))
+* add FormGroup component ([#809](https://github.com/chanzuckerberg/edu-design-system/issues/809)) ([3d48a2e](https://github.com/chanzuckerberg/edu-design-system/commit/3d48a2e2ea6d08fa88297e09ebfe3d5ee10b48f6))
+* remove unused legacy color tokens ([a953e25](https://github.com/chanzuckerberg/edu-design-system/commit/a953e256906dfee2d66cf46a9211a9149f80f29d))
+* use ts declarations format ([b0610ea](https://github.com/chanzuckerberg/edu-design-system/commit/b0610ea7f5331358a65265ff6117b17982957ccc))
+
+
+
+
+
```

Comment:
not sure if we commit the alpha releases, can revert

Reply:
If the `alpha` was only for testing the change, I think reverting makes sense. Could imagine keeping `alpha` releases around if we're building upon a big branch or something